Updated the Use case documentation to fix errors.

ALHennings opened the pull request:
Included but not limited to spelling, grammar, and formatting.

Miss named the branch slightly, This is regarding the documentation not the diagram.

Closes issue #32

ALHennings committed:
Updated the Use case documentation to fix errors. 

Included but not limited to spelling, grammar, and formatting.

tking2096 reviewed (changes requested):
There is now an empty page in the middle of the document.

Commit:
Shifted Unregistered User to the top.

Commit:
Removed blank page. Various other changes.

Adam, Alexander, and Cameron made changes to preconditions and flow for various use cases.
